pkgs/development/web/kore/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchFromGitHub, openssl }:
+
+stdenv.mkDerivation {
+  name = "kore-2.0.0";
+
+  src = fetchFromGitHub {
+    owner = "jorisvink";
+    repo = "kore";
+    rev = "a16348d5241b0b82f09c51cf3161872b6916cd76";
```

Comment:
`rev = "2.0.0-release";`

Reply:
Didn't realize GitHub supported `owner/repo/archive/tagname-release.format`. I'll go change that now.

Reply:
Actually the upstream tagname itself *is* "2.0.0-release", so it's just referring to a git tag, not any GitHub format or anything.